### ISSUE TITLE
[Feature Request] VDatePicker Range option

### ISSUE THREAD

**Issue by adamjimenez** [vuetifyjs/vuetify#18701]
### Problem to solve
Select a range of dates by selecting a start and end date.

### Proposed solution
This was working [in 3.3.18](https://play.vuetifyjs.com/#eNpVUMFugzAM/RUrFzqpEE29VBVF2j9Mu4wdIjBbVhIiE9AmlH+fSVhVTrbfs5+f3vsiRmrkPKHX3W/xPYqL0MYN5GGBhlB5fEscBOhoMJBtu1lta4s/cbUZ7OhhI+C6PzwstQXwX2jwArEHaLFTU+9fE5i1im7ZMVFSrjXwFJ5qK47R34tzBcuzudKjcT2rV+taOefKudjGgY14pS3SBjHoCCtS9hNhWYAghFKu0J2f85bVcqebGxLMuRla7K+1oFqAYY/a9QhRoCrlbvn/LaP7vyuSbJXywS6PY0PaeRjRT8n1PWvC7jFhThe2WIkDZfbAaZQyCfCpCEdxKk7FmRNa6/NZfPwB+pmZQQ==) but appears to have been dropped. I'm hoping it can come back.

**Comment by themegazord** [comment]
I noticed the lack of this method too. I'm thinking about using an ascending `sort` in the array that will receive the data from the `v-date-picker` and take the first and last data, until this feature is finished.

```html
<v-date-picker multiple v-model="data"></v-date-picker>
<v-btn @click="console.log(data.sort((a, b) =>{
  return a - b
}))"></v-btn>
```

```js
  data() {
    return {
      data: []
    }
  },
```

**Comment by kuba13666** [comment]
Can you maybe limit the number of options in multiple choice date picker? If you could do that that would give you a little bit similar experience if you limited it to two choices only as a workaround?

**Comment by TheSynt4x** [comment]
> Can you maybe limit the number of options in multiple choice date picker? If you could do that that would give you a little bit similar experience if you limited it to two choices only as a workaround?

Yeah, something like `:multiple="2"` would be great. Right now, there is no option for that and with that you cannot even limit the amount of choices like it'd work with a range.

**Comment by johnleider** [comment]
> Can you maybe limit the number of options in multiple choice date picker? If you could do that that would give you a little bit similar experience if you limited it to two choices only as a workaround?

Yes, but not with range.

**Comment by Rubon72** [comment]
why its closed? In my project user can select as much range as he want, if it's 3years, it will be array with 1000 element. Why just don't use "from" and "to" like in vuetify 2?  why you make it worse?

**Comment by johnleider** [comment]
Reopening because we need to do a better job of handling large ranges.

**Comment by KaelWD** [comment]
`multiple="range"` since 07781b385c0712baa8ec1974b11bd380d3548860 in 3.5.0
